src/parser/main.js

```
@@ -50,4 +50,132 @@ function replace(str, replacements) {
+    }
+
+    // Recursively render a command plus any piped successors.
+    function renderCommand(cmd) {
+        let parts = [];
+
+        // Component (executable name)
+        parts.push(renderParam(cmd.component));
+
+        // Parameters
+        if (cmd.params && cmd.params.length) {
+            cmd.params.forEach(param => {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)